https://issues.redhat.com/browse/ACM-30402 (#8647)
open-cluster-management/rhacm-docs: release_notes/acm_new.adoc

- The `RightSizingRecommendation` feature to optimize workloads is now generally available. After you enable the observability service, namespace right-sizing and virtualization right-sizing are enabled by default. To learn more, see link:../observability/obs_right_size_intro.adoc#right-size-intro[`RightSizingRecommendation` guides].

- You can now configure resource requirements for observability using the `AddonDeploymentConfig` resource when observability is enabled through the multicluster observability add-on. For more information, see link:../observability/obs_enable_mcoa.adoc#enable-mcoa[Enabling the multicluster observability add-on] and link:../add-ons/configure_klusterlet_addons.adoc#setting-addondeploymentconfig-klusterlet-addons[Setting up the AddOnDeploymentConfig to configure klusterlet add-ons].

> - The maximum compaction level for the Thanos compactor is `4`. This configuration enables the creation of one-hour downsampled blocks. As a result, the `observability-thanos-compact-0` pod uses more memory, and storage consumption in the S3 bucket increases. See, link:../observability/observability_arch.adoc#observability-arch[Observability architecture].

Reviewer: NIT it is true that storage consumption is going to increase, but it is also important to note that bucket usage will increase. The thanos compactor will do more PUT and DELETE operations to create blocks with a larger duration.

> - The maximum compaction level for the Thanos compactor is `4`. This configuration enables the creation of one-hour downsampled blocks. As a result, the `observability-thanos-compact-0` pod uses more memory, and storage consumption in the S3 bucket increases. See, link:../observability/observability_arch.adoc#observability-arch[Observability architecture].

Author: ```suggestion
- The maximum compaction level for the Thanos compactor is `4`. This configuration enables the creation of one-hour downsampled blocks. As a result, memory usage for the `observability-thanos-compact-0` pod and storage consumption in the object storage bucket increase. See, link:../observability/observability_arch.adoc#observability-arch[Observability architecture].
```

> - The maximum compaction level for the Thanos compactor is `4`. This configuration enables the creation of one-hour downsampled blocks. As a result, memory usage for the `observability-thanos-compact-0` pod and storage consumption in the object storage bucket increase. See, link:../observability/observability_arch.adoc#observability-arch[Observability architecture].

Author: Received some feedback from doc team and we are on the same page about having the Thanos operation details in the entry

```suggestion
- The maximum compaction level for the Thanos compactor is `4`. This configuration enables the creation of one-hour downsampled blocks. As a result, memory usage for the `observability-thanos-compact-0` pod and storage consumption in the object storage bucket increase.  The Thanos compactor completes more `PUT` and `DELETE` operations to create blocks with a larger duration. See, link:../observability/observability_arch.adoc#observability-arch[Observability architecture].
```

> - The maximum compaction level for the Thanos compactor is `4`. This configuration enables the creation of one-hour downsampled blocks. As a result, memory usage for the `observability-thanos-compact-0` pod and storage consumption in the object storage bucket increase.  The Thanos compactor completes more `PUT` and `DELETE` operations to create blocks with a larger duration. See, link:../observability/observability_arch.adoc#observability-arch[Observability architecture].

Reviewer 2: ```suggestion
- The maximum compaction level for the Thanos compactor is `4`. This configuration allows you to create one-hour downsampled blocks. As a result, memory usage for the `observability-thanos-compact-0` pod and storage consumption in the object storage bucket increase. The Thanos compactor completes more `PUT` and `DELETE` operations to create blocks with a larger duration. See, link:../observability/observability_arch.adoc#observability-arch[Observability architecture].
```

Just a suggestion for rewording and to avoid the use of "enable".

<img width="990" height="251" alt="Image" src="https://github.com/user-attachments/assets/8b3e802d-8e0a-467e-affc-6a3103378517" />

Author: Great call out! I agree with you and will apply your suggestion
